cfgov/unprocessed/css/organisms/form-explainer.less

```diff
@@ -0,0 +1,284 @@
+/* ==========================================================================
```

Comment:
This file is copied from [owning a home form explainer less](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/unprocessed/apps/owning-a-home/css/form-explainer.less), Changes include:

- removal of [tab styles](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/unprocessed/apps/owning-a-home/css/form-explainer.less#L211-L249)
- numbered (`.o-expandable__form-explainer-category-1`, `.image-map_overlay__category-1`) instead of named (`.o-expandable__form-explainer-checklist`, `.image-map_overlay__checklist`) category classes